src/Test/L0/Listener/Configuration/LinuxServiceControlManagerL0.cs

```diff
@@ -106,7 +106,7 @@ public void VerifyLinuxServiceConfigurationFileContent()
 [Service]
 ExecStart={0}/runsvc.sh
 User={1}
-Environment=NODE_ENV=production
+Environment=
```

Comment:
Should we just remove the line or leave it with no value?  Is there any distinction or recommendation in system?